tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Translation/translation.spec.ts

```diff
@@ -23,7 +26,53 @@ test.describe('Translation', () => {
     // Assert
     await umbracoUi.isSuccessNotificationVisible();
 
+  });
+  test('Create a 2nd dictionary key and filter the keys', async ({umbracoUi, umbracoApi, page}) => {
```

Comment:
Looking at this test I think you are assuming that the test above has run before, and that is not something you can actually rely on.
Example: you can run tests in parallel, or even individually, so it's good practice to make all tests independent of each other.
So in this case I think you could add a cleanup step at the end of the first test, and then in this one create 2+ new dictionary entries and use those.

Reply:
Thanks for all of your suggestions @lauraneto! All make good sense to me (and not sure why I didn't think of some...) I will put in some updated when I get a chance 😄